how-to-docs/drone-how-to.md

```diff
@@ -356,29 +356,39 @@ The build will fail with the following error:
 [ERROR] 2016/10/20 10:17:40 main.go:130: exit status 1
 ```
 
-This suggests that you are not authorised to deploy to the kubernetes cluster. You can fix this by setting your `KUBE_TOKEN` to a valid value. But before you do this, let's recap the environment variables needed to deploy successfully to DSP.
+This suggests that you are not authorised to deploy to the kubernetes cluster. You can fix this by setting your `KUBE_TOKEN` to a valid value. But before you do this, let's recap the environment variables needed to deploy successfully to ACP.
+
+The kube `deploy.sh` script relies on 3 environment variables:
```

Comment:
Might need to reword / change the way you're referring to this deploy script – it's something the end user creates, manages and customises for their particular scenario. So we should provide examples, guidance and best practices, but leave it at that.

(Right now it sounds like a tool that they get out of the box).

Reply:
I think this part (just the part about the error) would be better suited in Q&As.

Reply:
I think this was originally put here because it's so likely to occur, and so it's useful to have as part of the learning flow (as opposed to the troubleshooting flow).